frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -577,8 +584,11 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
                                             data-attr="edit-feature-flag"
                                             type="secondary"
                                             disabledReason={
-                                                !featureFlag.can_edit &&
-                                                "You have only 'View' access for this feature flag. To make changes, please contact the flag's creator."
+                                                !featureFlag.can_edit
+                                                    ? "You have only 'View' access for this feature flag. To make changes, please contact the flag's creator."
+                                                    : featureFlag.deleted
+                                                    ? 'This feature flag has been deleted. Restore it to edit.'
+                                                    : null
```

Comment:
Not sure if I'm doing something wrong, but I'm not seeing `Restore Feature Flag` in red when I try this on my branch
<img width="1920" alt="image" src="https://github.com/user-attachments/assets/eaea6a24-ca42-4236-8b5e-98b6100f5cd9">

Reply:
Ah whoops, I changed it to be default text after I recorded the loom, my b! Think it should be danger text either way?

Reply:
Reverted this to be danger text always, feels appropriate